optimum/habana/transformers/trainer.py

```diff
@@ -26,6 +26,7 @@
 
 import numpy as np
 import torch
+from torch import nn
```

Comment:
Not a big fan of importing `nn`, let's just use `torch.nn` below

Reply:
done